README.md

```diff
@@ -7,11 +7,11 @@
 [SimpleLogin](https://simplelogin.io) | Privacy-First Email Forwarding and Identity Provider Service
 ---
 <p>
-<a href="https://chrome.google.com/webstore/detail/simplelogin-protect-your/dphilobhebphkdjbpfohgikllaljmgbn">
+<a href="https://chrome.google.com/webstore/detail/dphilobhebphkdjbpfohgikllaljmgbn">
```

Comment:
Do you know what is the difference between these 2 urls? The new one looks definitely shorter and more correct 👍

Reply:
It just doesn't include the name of the extension. Usually it gets included to make it more user friendly. But for this use case I think it makes more sense because you could change the name of the extension.